CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 
 ### Features Added
 - Aws bulk loading for secp256k1 keys in eth1 mode [#889](https://github.com/Consensys/web3signer/pull/889)
+- Add High Watermark functionality [#696](https://github.com/Consensys/web3signer/issues/696) 
```

Comment:
Can you also add a note for the breaking change in the watermark repair now we don't support specifying individual validators

Reply:
That's already merged https://github.com/Consensys/web3signer/commit/a383c3287d6a1bdf5fb8112c3729574c91a5b792